fixtures/questions/create_questions_fixtures.py

```diff
@@ -164,3 +164,15 @@
         ]
     }
 }
+
+query_update_total_views = '''
```

Comment:
Kindly use a more descrip[tive variable names. With this name one cannot tell 'query_update_total_views' is the total views for what exactly.